src/Validator/Formatter.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Overblog\GraphQLBundle\Validator;
+
+use Overblog\GraphQLBundle\Event\ErrorFormattingEvent;
+use Overblog\GraphQLBundle\Validator\Exception\ArgumentsValidationException;
+
+class Formatter
+{
+    public function onErrorFormatting(ErrorFormattingEvent $event): void
```

Comment:
Can you add a comment with an example of the output of errors after formatting please?

Reply:
Comment to the pho or just here?

Reply:
to php please

Reply:
I added a [section](https://github.com/murtukov/GraphQLBundle/blob/feature/validation_v.0.13/docs/validation/index.md#error-messages) to the documentation and added a @link annotation to the php, since the inline example would be too large. 
Also response examples could be found under the path: `GraphQLBundle\tests\Functional\Validator\ExpectedErrors.php`.